src/utils.ts

```diff
@@ -53,7 +43,13 @@ export function isAnyoneCanPay(script: Script | undefined, config: SporeConfig<s
     return false;
   }
   if (isOmnilockScript(script, config)) {
-    return script.args.slice(44, 46) === '02';
+    const flag = number.Uint8.unpack(`0x${script.args.slice(44, 46)}`);
```

Comment:
I guess here needs more information for a better explanation

Reply:
Added comment